selfdrive/debug/internal/measure_modeld_packet_drop.py

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python3
 import cereal.messaging as messaging
+from typing import Optional
 
 if __name__ == "__main__":
   modeld_sock = messaging.sub_sock("modelV2")
 
   last_frame_id = None
-  start_t = None
+  start_t:Optional[int] = None
```

Comment:
Generally there's a space after the colon for types

Reply:
Thanks, haven't noticed any typed variables so wasn't really sure how to proceed.